src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java

```diff
@@ -25,12 +25,16 @@
      * Serialize an object into a JSON string representation.
      */
     public static String toJson(Object object) {
+
+        String message;
         try {
             return mapper.writeValueAsString(object);
         } catch (JsonProcessingException e) {
             e.printStackTrace();
+            message = e.getMessage();
         }
-        return "{}";
+
+        return String.format("{Error: Unable to serialize object: %s. %s.", object.toString(), message);
```

Comment:
This is invalid JSON. Ideally, we should just make a call to `JsonUtils#formatJSON`. See https://github.com/ibi-group/otp-middleware/blob/a323b679905673d781a19358acf679baad4ac3dd/src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java#L98-L100